tests/utils.js

```diff
@@ -651,6 +651,7 @@ const dockerComposeCmd = (...params) => {
     DB1_DATA: db1Data,
     DB2_DATA: db2Data,
     DB3_DATA: db3Data,
+    COUCHDB_SECRET: 'monkey',
```

Comment:
I am trying to understand what setting this does.  Am I correct in thinking that this is overwriting the default envar set in the [docker-compose file](https://github.com/medic/cht-core/blob/bc54098b0aad7397142cf762b7b1a865700c6fdb/couchdb/test.couchdb-cluster.yml#L14)?  

Do we want to leave this set as the short `monkey` value or do you think it would be better to set a more proper example value (or even just remove this at let it continue to use the default from the compose file)?

Reply:
The compose file doesn't have a default setting, it was removed a few commits ago.

Reply:
> The compose file doesn't have a default setting, it was removed a few commits ago.

:confused: I see that the default password was removed, but it looks like there is still a [default `COUCHDB_SECRET`](https://github.com/medic/cht-core/blob/6e77c909bc72d4e2f11aadbf9d0cea4593f6c95e/couchdb/test.couchdb-cluster.yml#L14).... (But maybe I am misunderstanding something here!)